data.sql

```diff
@@ -38,4 +38,55 @@ VALUES
 (8,'Angemon','2005-06-12',45,true,1),
 (9,'Boarmon','2005-06-07',20.4,true,7),
 (10,'Blossom','1998-10-13',17,true,3),
-(11,'Ditto','2002-05-14',22,true,4);
\ No newline at end of file
+(11,'Ditto','2002-05-14',22,true,4);
+
+-- DAY 3
+
+-- INSER DATA IN owners TABLE
+
+INSERT INTO owners 
+VALUES
+(DEFAULT,'Sam Smith',34),
+(DEFAULT,'Jennifer Orwell',19),
+(DEFAULT,'Bob',45),
+(DEFAULT,'Melody Pond',77),
+(DEFAULT,'Dean Winchester',14),
+(DEFAULT,'Jodie Whittaker',38);
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add descriptive data to `owners` => `INSERT INTO` and `VALUES` like a below:
```sql
 INSERT INTO owners (full_name, age)
 VALUES ('Sam Smith', 34);

 INSERT INTO owners (full_name, age)
 VALUES ('Jennifer Orwell', 19);

 INSERT INTO owners (full_name, age)
 VALUES ('Bob', 45);

 INSERT INTO owners (full_name, age)
 VALUES ('Melody Pond', 77);

 INSERT INTO owners (full_name, age)
 VALUES ('Dean Winchester', 14);

 INSERT INTO owners (full_name, age)
 VALUES ('Jodie Whittaker', 38);
```

Reply:
Hi @mukhammadrizooff ,
Thank you for the review.
But I think what I did is the same thing as what you are recommending.
If data is being added in the same sequence as the columns were created, there is no need to specify the column names.
And when adding multiple data to a table, I don't think there is any need to add them one after the other when they can all be added at once.
Please, kindly review again Sir.
Regards.

Reply:
Hi@Alhajideen! First of all, I didn't say that what you did was the wrong bro, but the reason why the method you wrote will not be clean was not specified, so I gave an example of how to write the method above. I believe this will be useful for you in your future work. I'm sorry if I offended you and I wish you luck 😃

Reply:
@mukhammadrizooff ,
You are so kind. Thank you very much.
Trust me, I am not offended, I like when changes are requested on my PR's.
Thanks for calling and explaining everything, I have implemented the changes just as you requested.